src/components/PopUpTrackInfo.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import PropTypes from "prop-types";
+import PopupDialog from './PopupDialog.js';
+
+export const PopUpTrackInfo = ({
+  open,
+  attributes,
+  close,
+}) => {
+    return(
+      <div>
+        <PopupDialog open={open} close={close}>
+            <h5>Object Information</h5>
+            <table>
+              {/* Node info here */}
+              {(attributes || []).map(function(attribute){
+                return <tr key={attribute[0]} style={{border: '1px solid black'}}>
+                  <td style={{border: '1px solid black', fontWeight: 'bold'}}>{attribute[0]}</td>
```

Comment:
Having a border on the row and also on one cell is a little weird, but it seems to look right.